Mathlib/Computability/TMComputable.lean

```diff
@@ -43,14 +43,22 @@ namespace Turing
 the namespace `Turing.TM2` in `TuringMachine.lean`), with an input and output stack,
  a main function, an initial state and some finiteness guarantees. -/
 structure FinTM2 where
-  {K : Type} [kDecidableEq : DecidableEq K] [kFin : Fintype K] -- index type of stacks
-  (k₀ k₁ : K) -- input and output stack
-  (Γ : K → Type) -- type of stack elements
+  /-- Index type of stacks -/
+  {K : Type} [kDecidableEq : DecidableEq K] [kFin : Fintype K]
```

Comment:
This might label all three items with the same docstring (for `(implicit args)`, I verified it does).
This is not desirable here - can you break the line, and perhaps even document the other arguments. See #12480 for an idea with what.